From: Modestas Vainius <modax@debian.org>
Subject: Make FindGTK2.cmake work with Debian multiarch
Bug: http://www.cmake.org/Bug/view.php?id=12049
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=618932#24
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/cmake/+bug/751940
Origin: vendor
Forwarded: no
Last-Update: 2011-06-07

Fix FindGTK2.cmake to work with Debian/Ubuntu multiarch library paths. The
patch depends on multiarch_library_directory_hack.diff patch.

--- a/Modules/FindGTK2.cmake
+++ b/Modules/FindGTK2.cmake
@@ -185,6 +185,7 @@ function(_GTK2_FIND_INCLUDE_DIR _var _hd
 
     find_path(${_var} ${_hdr}
         PATHS
+            /usr/lib/${CMAKE_ARCH_TRIPLET}
             /usr/local/lib64
             /usr/local/lib
             /usr/lib64
